apps/web-giddh/src/app/subscription/subscription.component.scss

```diff
@@ -1,14 +1,51 @@
-.subscription-container {
-    max-width: 1360px;
+.wrap-tab-content-pd-15 {
+    padding: 0 15px;
+}
 
-    .giddh-table {
-        border:  1px solid var(--color-table-primary-content-bar-border) !important;
-    }
+.activity-title {
+    justify-content: space-between;
```

Comment:
use class

Reply:
do not change anything on old subscription module